### PR TITLE
`create_sibling_dataverse` -> `add_sibling_dataverse` 

### LINKED ISSUE #224: Replace `create_sibling_dataverse()` with `add_sibling_dataverse()`
The `create_sibling_dataverse` command is not in a good place. Some key issues are

- https://github.com/datalad/datalad-dataverse/issues/136
- https://github.com/datalad/datalad-dataverse/issues/139
- https://github.com/datalad/datalad-dataverse/issues/152

However, the biggest unresolved issue is metadata declaration for a dataset. Given that there is no readily implementable concept yet, and any `create` needs to be followed up with manual editing on the web UI, there is little point in having such a command.

The opportune alternative seems to be an `add-sibling-dataverse` command, that does the "same", but expects an existing dataverse dataset as a target to be specified.

This should not cause more manual labor (maybe even less for now), and makes sure that datasets are created according to the policies of a particular dataverse instance. This would remove the following issues from the "problem surface":

- https://github.com/datalad/datalad-dataverse/issues/193
- https://github.com/datalad/datalad-dataverse/issues/152
- https://github.com/datalad/datalad-dataverse/issues/139
- https://github.com/datalad/datalad-dataverse/issues/137
- https://github.com/datalad/datalad-dataverse/issues/186
- https://github.com/datalad/datalad-dataverse/issues/192
- https://github.com/datalad/datalad-dataverse/issues/136
- https://github.com/datalad/datalad-dataverse/issues/102

With this change in paradigm, the recursive-mode feature of `create_sibling_dataverse` has to be critically evaluated. It is a thing wrapper around `for-each-dataset`, and, as such, offers litte extra convenience. At the same time it prescribes a tight corset of dataset representation on dataverse.

I believe it should be removed.

As a replacement, `add_sibling_dataverse` should support a `prefix` option (`None` by default) that enables users to identify a root `directoryLabel` for a particular DataLad dataset inside a Dataverse dataset. This makes it possible to represent any number of DataLad datasets in a single Dataverse dataset. This matches the need to create the Dataverse dataset manually (would be a one-time effort), and does not constrain automation of data deposition with large DataLad dataset hierarchies. Providing `prefix` rather than `--recursive` enables users to choose their own organization of subdatasets within the Dataverse dataset, rather than imposing a select few. Recursive operation can be done conveniently via `for_each_dataset`.

### PR DESCRIPTION
**Pull request by mih**
Closes #224

Sits on top of #234, because too much stuff going on on appveyor https://github.com/datalad/datalad/issues/7324

### CODE REVIEW

File: datalad_dataverse/add_sibling_dataverse.py [hunk 1]
```diff
@@ -78,18 +67,23 @@ class CreateSiblingDataverse(Interface):
         dict(text="Create a dataverse dataset sibling for sharing and citing",
              code_py="""\
                  > ds = Dataset('.')
-                 > ds.create_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
+                 > ds.add_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
              """,
              code_cmd="datalad create-sibling-dataverse demo.dataverse.org -s dataverse",
```
Comment (adswa) [inline datalad_dataverse/add_sibling_dataverse.py:72]:
```suggestion
             code_cmd="datalad add-sibling-dataverse demo.dataverse.org -s dataverse",
```

File: datalad_dataverse/__init__.py [hunk 2]
```diff
@@ -15,13 +15,9 @@
...
             # importable module that contains the command implementation
-            'datalad_dataverse.create_sibling_dataverse',
+            'datalad_dataverse.add_sibling_dataverse',
             # name of the command class implementation in above module
-            'CreateSiblingDataverse',
-            # optional name of the command in the cmdline API
-            'create-sibling-dataverse',
-            # optional name of the command in the Python API
-            'create_sibling_dataverse'
+            'AddSiblingDataverse',
```
Comment (adswa) [inline datalad_dataverse/__init__.py:20]: This needs the other entries, too, in order to build the manpages
```suggestion
            'AddSiblingDataverse',
            # optional name of the command in the cmdline API
            'add-sibling-dataverse',
            # optional name of the command in the Python API
            'add_sibling_dataverse'
```

Comment (mih) [reply datalad_dataverse/__init__.py:20]: Thx! I edited the suggestion for clarity.

File: datalad_dataverse/add_sibling_dataverse.py (hunk 1 above)
Comment (mih) [reply datalad_dataverse/add_sibling_dataverse.py:72]: Thx, I will wait a little with these, and see where the API ends up.

**Comment by mih** [comment]
I will merge this here, to be able to get the test update into https://github.com/datalad/datalad-dataverse/pull/237

**Pull request merged**